### PR TITLE
Added Shortcodes for Tweets, Weather, Calender, Title, News

### PR DESCRIPTION
**Pull request by chasethenag420**
Added required shortcodes with their functionality and still some refactoring needs to be done

### CODE REVIEW

**Comment by idmontie** [comment]
@chasethenag420 You need to make a localsettings.php file for travis in the travis file. We have done this in our Gios2-PHP repo, so you can use that as an example.  The files you should look at are the `.travis.yml` file and `tests/bootstrap`.

File: assets/js/kiosk-helper.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/*! kisok v1.0 */
+function scrollTweets() {
+     $('#kiosk_tweets_list').animate({
```
Comment (rraub) [inline assets/js/kiosk-helper.js:3]: Shouldn't this be using the class selector rather than the id selector?

File: kiosk-plugin.php [hunk 2]
```diff
@@ -32,16 +32,21 @@ function setup_kiosk_wp_plugin() {
 
   // Verify kiosk api version
   // Mo version checking for now
-
+  // Require all the files for the Gios_Api
```
Comment (rraub) [inline kiosk-plugin.php]: Gios_Api? do you mean the kiosk plugin?

File: localsettings.default.php [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?php // Copy to localsettings.php to be included
+
+$localsettings['twitter_handle']              = 'YOUR_DB_NAME_HERE';
```
Comment (rraub) [inline localsettings.default.php]: When you bootstrap the localsettings.php for travis you'll want to change the default values for these settings so they are unique and match the setting key.

**Pull request merged**